bundle/direct/bundle_apply.go

```diff
@@ -75,6 +80,10 @@ func (b *DeploymentBundle) Apply(ctx context.Context, client *databricks.Workspa
 		}
 
 		if at == deployplan.ActionTypeDelete {
+			if migrateMode {
+				logdiag.LogError(ctx, fmt.Errorf("%s: Unexpected delete action during migration", errorPrefix))
```

Comment:
This error should be more actionable; e.g. revert to previous configuration, or run a non-migration deploy first to achieve a clean slate, etc.